src/chapters/thymeleaf-views/exercises.rst

```diff
@@ -0,0 +1,15 @@
+Exercises: Thymeleaf
+=====================
+
+In the chapter, we started working on an application for tracking various coding events around town.
+
+Open up your ``codingevents`` project in IntelliJ.
+
+#. In addition to names of events, we want to add descriptions as well.
+   Use fragments to store the name and description of different coding events around town.
```

Comment:
How about using a fragment to store the generic HTML structure for name/description, and then pull this structure into the different views? I feel that storing each event as a fragment would be more unwieldy than using ${event.name} and ${event.description} in a single pair of HTML tags.

What about fragments for some new feature that appears on each page?  Maybe a "Last Updated" or "Created By" footer, a decorative border, or favorite links?

Reply:
Since they haven't learned models, I don't think the structure of using an ``event`` object because it is difficult to store without knowledge of models. There are also only 4 or so events used in the video. The fragments don't have to be about the events. Though maybe using fragments for event descriptions is better suited for a bonus mission. Food for thought!

Reply:
Also, it turns out they will make a fragment for the header/footer in the videos!